app/main/views/login.py

```diff
@@ -168,8 +167,8 @@ def update_password(token):
     if form.validate():
         if data_api_client.update_user_password(user_id, password, email_address):
             current_app.logger.info(
-                "User {user_id} successfully changed their password",
-                extra={'user_id': user_id})
+                "User {user_id} ({hashed_email}) successfully changed their password",
```

Comment:
i feel like the hashed email is pointless, if not dangerous because it will then be stored in logging systems and archived.  Out internal user id would be enough to identify user.